packages/dts-gen/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "dts-gen",
+  "version": "0.7.3",
```

Comment:
It should, but I'll bump it to 0.7.4.

Right now on merge and release, it'll go to 0.8.0, though. We could do a major bump if wanted; since it's zero versioned, there's no difference.